### PR TITLE
[fix] Fix NPE when getting map params

### LINKED ISSUE #260: [Bug] release-1.5.0 will cause null point exception when the params without --table-conf.
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-doris/issues?q=is%3Aissue) and found no similar issues.

### Version

release-1.5.0

### What's Wrong?

image
image
image

### What You Expected?

in the old version. it has do check null at build function, in the latest version, it is need to do that.

### How to Reproduce?

_No response_

### Anything Else?

_No response_

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by murong00**
# Proposed changes

close #260

## Problem Summary:

Return a new map instead of null when the key-value parameters are invalid or not specified.

## Checklist(Required)

1. Does it affect the original behavior: (No
2. Has unit tests been added: (No Need)
3. Has document been added or modified: (No Need)
4. Does it need to update dependencies: (No)
5. Are there any changes that cannot be rolled back: (No)

### CODE REVIEW

File: flink-doris-connector/src/main/java/org/apache/doris/flink/tools/cdc/CdcTools.java [hunk 1]
```diff
@@ -148,7 +148,7 @@ private static void syncDatabase(
 
     private static Map<String, String> getConfigMap(MultipleParameterTool params, String key) {
         if (!params.has(key)) {
-            return null;
+            return new HashMap<>();
```
Comment (JNSimba) [inline flink-doris-connector/src/main/java/org/apache/doris/flink/tools/cdc/CdcTools.java:151]: Is there any problem with returning null?

Comment (JNSimba) [reply flink-doris-connector/src/main/java/org/apache/doris/flink/tools/cdc/CdcTools.java:151]: Here is the initialization of tableConfig  https://github.com/apache/doris-flink-connector/blob/master/flink-doris-connector/src/main/java/org/apache/doris/flink/tools/cdc/DatabaseSync.java#L65

Comment (murong00) [reply flink-doris-connector/src/main/java/org/apache/doris/flink/tools/cdc/CdcTools.java:151]: However the initialization value was reset to `null` on below:
https://github.com/apache/doris-flink-connector/blob/d49eda9c0f766b9ee2561da850a3a137c5ad7c5a/flink-doris-connector/src/main/java/org/apache/doris/flink/tools/cdc/CdcTools.java#L134
You can just run `CdcTools` to reproduce this without `--table-conf` parameter.

**Review by JNSimba — APPROVED** [review]
LGTM

File: flink-doris-connector/src/main/java/org/apache/doris/flink/tools/cdc/CdcTools.java (hunk 1 above)
Comment (JNSimba) [reply flink-doris-connector/src/main/java/org/apache/doris/flink/tools/cdc/CdcTools.java:151]: Yes, it's possible

**Pull request merged**